hack/import-restrictions.yaml

```diff
@@ -135,6 +135,20 @@
   - k8s.io/component-base
   - k8s.io/klog
 
+- baseImportPath: "./vendor/k8s.io/apiextensions-apiserver/pkg/controller/openapi/"
+  allowedImports:
+  - k8s.io/apiextensions-apiserver
+  - k8s.io/apimachinery
+  - k8s.io/apiserver
+  - k8s.io/client-go
+  - k8s.io/klog
+  - k8s.io/kube-openapi
+  # TODO(roycaihw): the following two should be tightened, either by
+  # k8s.io/apiextensions-apiserver generating necessary openapi through Makefile,
+  # or by k8s.io/api publishing generated openapi go files
+  - k8s.io/kubernetes/pkg/generated/openapi
+  - k8s.io/kubernetes/pkg/api/legacyscheme
```

Comment:
we can't allow these. Dependencies back to the k/k are forbidden as they break publishing.